docs/using-connector.md

```diff
@@ -533,6 +534,160 @@ conf.set("spark.riakts.write.bulk-size", "500")
+      StructField(name = "pressure", dataType = DoubleType)))
+
+    val sparkConf = new SparkConf(true)
+      .setAppName("Simple Spark Streaming to Riak TS Demo")
+
+    setSparkOpt(sparkConf, "spark.master", "local")
+    setSparkOpt(sparkConf, "spark.riak.connection.host", "127.0.0.1:8087")
+    setSparkOpt(sparkConf, "kafka.broker", "127.0.0.1:9092")
+
+    val sc = new SparkContext(sparkConf)
+    val streamCtx = new StreamingContext(sc, Durations.seconds(15))
+    val sqlContext = new org.apache.spark.sql.SQLContext(sc)
+    import sqlContext.implicits._
+
+    val kafkaProps = Map[String, String](
+      "metadata.broker.list" -> sparkConf.get("kafka.broker"),
+      "client.id" -> UUID.randomUUID().toString
+    )
+```
+
+This first section of code just sets up the table schema, a spark streaming context, a spark sql context, and Kafka properties. Note that we need to set up a table in TS that reflect the schema in the previous section of code. We can create this table in TS with:
+
+```
+curl -v -XPUT -H 'Content-Type: application/json' "http://$RIAK_HTTP/admin/explore/clusters/default/bucket_types/ts_weather_demo" -d '{"props":{"n_val":3, "table_def":"CREATE TABLE ts_weather_demo (weather varchar not null,family varchar not null,time timestamp not null,temperature  double,humidity double,pressure double,PRIMARY KEY ((weather, family, quantum(time, 1, 'h')), weather, family, time))"}}'
+```
+
+Be sure to substitute the Riak's IP address and http port in for `$RIAK_HTTP`.
+
+The next section of the code is:
+
+```scala
+    KafkaUtils
+      .createDirectStream[String, String, StringDecoder, StringDecoder](streamCtx, kafkaProps,
+      Set[String]("streaming"))
+      .foreachRDD { rdd => rdd.map(println)
+        val rows = sqlContext.read.schema(schema).json(rdd.values)
+          .withColumn("time", 'time.cast("Timestamp"))
+          .select("weather", "family", "time", "temperature", "humidity", "pressure")
+
+        rows.write
+          .format("org.apache.spark.sql.riak")
+          .mode(SaveMode.Append)
+          .save("ts_weather_demo")
+      }
+```
+In this section of code, we are setting up a stream from Kafak topic `streaming` into TS table `ts_weather_demo`. Here we are using our spark sql context to read each rdd streamed from the kafka topic and then write into the TS table.
+
+Now that we have seen the code lets run the example (see [here](./building-and-testing-connector.md#build) if you need to build the spark-riak-connector first). You can run the `StreamingTSExample.scala` example, after building, with:
+
+ ```
+ /path/to/spark-riak-connector-examples/bin/run-example streaming.StreamingTSExample
+ ```
+
+Now that the stream is up and running, we need to actually send data to the Kafka topic. Lets start `the kafka console producer and send a chunk of data to it with:
+
+ ```
+ /path/to/bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streaming
+ ```
+ 
+Then the data:
```

Comment:
Please add more detail about how to send the JSON message to the console producer, as above